### DISCUSSION TITLE
Dual monitor fullscreen

### DISCUSSION THREAD

**Discussion by mmol67** [Xpra-org/xpra#3894] [Q&A]
Hello. Archlinux server (VM) and client. Version xpra v4.4.5-r0

Launching Xpra like this:

`xpra monitor :18 --start=xfce4-session --resize-display=1910x1000,1910x1000` or using 1920x1080,1920x1080 (native monitors resolution in the client)

client:` xpra attach ssh://miguel@192.168.100.61:2203/18` or using `--desktop-fullscreen`

A strange "translation" effect when going from windows mode to fullscreen mode and vice-versa. This "translation" is variable, it set to white of a copy of the image in a region and fixes it in the lower part of the window.
![Screenshot_3](https://github.com/Xpra-org/xpra/assets/60692795/68942396-6563-4c04-8519-a056741c2e87)

1) If I use --desktop-fullscreen both windows are drawing in the same monitor, so I have to shift-F11, move windows and shift-F11 again, this thing happens.
2) if I don't use fullscreen, move the windows and then shift-F11 SOMETIMES the "translation" happens.

My questions:

Is this a bug?
Is there a way to force each window fullscreen  in a monitor when using --desktop-fullscreen so I can just connect and let it work without having to use shif-F11 to be able to move windows (avoid shift-F11 effect)?

EDIT: anotated fullscreen capture
![Screenshot_4](https://github.com/Xpra-org/xpra/assets/60692795/5bc5a6a2-d1f4-4cdc-873c-a7b4bba434e7)

The client is KDE, I've tried two VM as server, one LXQT and the other XFCE as DEs

EDIT2: The main display could be "recovered" just clicking in the empty areas. Is the second display lost space what seems not "recoverable". And it has something to do with the height of the main window. If you keep it less of the half total monitor heigh, second window draws ok, no matter in window mode or fullscreen. If you go fullscreen or make the main window higher the second window get this effect proportonally. So main window go fullscreen, second window lose half vertical space, the botton border goes to the half of the screen.

**Comment by totaam** [comment]
Moved to #3920
